extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/querykit/BaseLeafFrameProcessorFactory.java

```diff
@@ -126,11 +124,12 @@ public ProcessorsAndChannels<Object, Long> makeProcessors(
       final OutputChannel outputChannel = outputChannelFactory.openChannel(0 /* Partition number doesn't matter */);
       outputChannels.add(outputChannel);
       channelQueue.add(outputChannel.getWritableChannel());
-      frameWriterFactoryQueue.add(stageDefinition.createFrameWriterFactory(outputChannel.getFrameMemoryAllocator(), removeNullBytes));
+      frameWriterFactoryQueue.add(stageDefinition.createFrameWriterFactory(
+          outputChannel.getFrameMemoryAllocator(),
```

Comment:
I think there are some formatting changes, could we please adjust the formatter so that we can see only the legit diff.